modules/tensor_mechanics/doc/content/source/auxkernels/RankTwoScalarAux.md

```diff
@@ -280,13 +280,36 @@ where $S_{ij}$ is the deviatoric tensor of the Rank-2 tensor $T_{ij}$.
 
 ## Volumetric Strain
 
-The scalar type `VolumetricStrain` computes the change in volume divided by the original volume and
-is most appliable for strain.  For a Rank-2 tensor, $T_{ij}$, the volumetric strain quantity is
-calculated with [eq:volumetric_strain_scalar_type].
+The scalar type `VolumetricStrain` computes the volumetric strain, defined as
 \begin{equation}
-\label{eq:volumetric_strain_scalar_type}
-s = T_{11} \cdot \left( T_{22} + T_{33} \right) + T_{22} \cdot T_{33} + T_{11} \cdot T_{22} \cdot T_{33}
+\label{eq:volumetric_strain}
+\varepsilon_{vol} = \frac{\Delta V}{V}
 \end{equation}
+where $\Delta V$ is the change in volume and $V$ is the original volume.
+
+This calculation assumes that the strains supplied as input ($T$) are logarithmic strains,
+which are by definition $log(L/L_0)$, where $L$ is the current length and $L_0$
+is the original length of a line segment in a given direction. 
+The ratio of the volume of a strained cube to the original volume is thus:
+\begin{equation}
+\label{eq:volumetric_strain_from_tensor}
+s = \exp(T_{11}) * \exp(T_{22}) * \exp(T_{33}) - 1
+\end{equation}
```

Comment:
This isn't the ratio of the volumes (V/V0) but delV/V0, right?

Reply:
Yes, that's right.  The equation just above this defines it as delta V / V.

Reply:
It looks like logarithmic volumetric strain (log (V/V0)) is a thing:
https://books.google.com/books?id=IDdJDwAAQBAJ&pg=PA434&lpg=PA434&dq=volumetric+strain+logarithmic&source=bl&ots=Me2UMrfQpX&sig=fUZUTh5oAgg54oOYE7CzjjusDuc&hl=en&sa=X&ved=2ahUKEwj1ttnYwazeAhUjc98KHaRMCGc4ChDoATAJegQIARAB#v=onepage&q=volumetric%20strain%20logarithmic&f=false
Maybe that is what we should be reporting to be consistent with the logarithmic strain tensor that we're computing?